src/engine/guidance/post_processing.cpp

```diff
@@ -44,6 +67,13 @@ RouteStep forwardInto(RouteStep destination, const RouteStep &source)
     return destination;
 }
 
+// invalidate a step and set its content to nothing
+inline void invalidateStep(RouteStep &step)
```

Comment:
This is a translation unit local helper function, right? Put it either in an anonymous namespace or make it static (instead of inline).